### PR TITLE
Improve error message for missing runtime errors

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun 

I never liked the obscure error message `undefined method 'errors' for nil` when there were runtime errors expected and they didn't happen. This fixes it.

### CODE REVIEW

**Comment by allewun** [comment]
:santa:

File: features/support/env.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@ def initialize_environment
 
 
 After do
-  expect(@last_run_result.error).to be_falsy, 'Expected no runtime error' unless @error_expected
+  expect(@last_run_result.try :error).to be_falsy, 'Expected no runtime error' unless @error_expected
```
Comment (charlierudolph) [inline features/support/env.rb]:
```ruby
if @last_run_result && !@error_expected
  expect(@last_run_result.error).to be_falsy, 'Expected no runtime error'
end
```

Comment (kevgo) [inline features/support/env.rb]: Agreed that  this reads better. Implemented

**Comment by kevgo** [comment]
All comments addressed

**Comment by charlierudolph** [comment]
:ship:

**Pull request closed without merging**